src/Shared/ServerInfrastructure/StringUtilities.cs

```diff
@@ -560,12 +561,14 @@ private static bool CheckBytesInAsciiRange(long check)
             return (((check - 0x0101010101010101L) | check) & HighBits) == 0;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
This didn't inline always in some constellations. So I'll force the JIT to do so :wink: